src/components/InformationsBar/CloroplethLegend/CloroplethLegend.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import 'jest-styled-components';
+import { render, screen } from '@testing-library/react';
+import { CloroplethLegend } from '.';
+import { Container, Title, Legend, GradientBar } from './styles';
+
+describe('CloroplethLegend', () => {
+
+    test('Test the Container component style in Choropleth Legend', () => {
+        const { container: container } = render(<Container />);
+        expect(container.firstChild).toHaveStyleRule('bottom', '100px');
+        expect(container.firstChild).toHaveStyleRule('position', 'fixed');
+        expect(container.firstChild).toHaveStyleRule('font-size', '14px');
+        expect(container.firstChild).toHaveStyleRule('width', '305px');
+        expect(container.firstChild).toHaveStyleRule('display', 'flex');
+        expect(container.firstChild).toHaveStyleRule('flex-direction', 'column');
+        expect(container.firstChild).toHaveStyleRule('border-radius', '5px !important');
+    });
+
+    test('Test the title of Choropletic Legend', () => {
+        render(<CloroplethLegend />);
+        const titleText = screen.getByText('População') as HTMLElement;
+        expect(titleText.textContent).toContain('População');
```

Comment:
```suggestion
        screen.getByText('População');
```

Reply:
Using the `expect` here is a double verification, the `getBy...` already have the verification and the throw error inside the function, you can validate this at the testing-library doc bellow:
https://testing-library.com/docs/queries/about/

Reply:
Using the `expect` here is a double verification, the `getBy...` already have the verification and the throw error inside the function, you can validate this at the testing-library doc bellow:
https://testing-library.com/docs/queries/about/